easybuild/easyconfigs/w/Wannier90/wannier90-3.1.0-gfort.patch

```diff
@@ -0,0 +1,24 @@
+--- config/make.inc.gfort.orig	2020-03-05 13:41:10.000000000 -0500
++++ config/make.inc.gfort	2022-06-02 09:22:27.141245863 -0400
+@@ -4,11 +4,11 @@
+ F90 = gfortran
+ 
+ #Add next two lines for parallel postw90. Choose the correct name of the mpi f90 wrapper
+-#COMMS  = mpi
+-#MPIF90 = mpgfortran #mpif90
++COMMS  = mpi
++MPIF90 = mpif90
```

Comment:
Use `$(MPIF90)` to make the patch more generic (i.e. take the value from the environment variable `MPIF90`